packages/skeleton-react/src/lib/components/Accordion/Accordion.tsx

```diff
@@ -46,111 +36,104 @@ const AccordionRoot: React.FC<AccordionProps> = ({
 	iconOpen = '-',
 	iconClosed = '+',
 	// Events
-	onValueChange = () => {},
+	onValueChange = noop,
 	// Children
-	children
+	children,
+	// Zag
+	...zagProps
 }) => {
-	// State
-	const [value, setValue] = useState<string[]>(multiple ? valueInit : [valueInit[0]]);
-
-	// Functions
-	function open(id: string) {
-		setValue((opened) => (multiple ? [...opened, id] : [id]));
-	}
-	function close(id: string) {
-		setValue((opened) => opened.filter((_id) => _id !== id));
-	}
-	function toggle(id: string) {
-		isOpen(id) ? close(id) : open(id);
-	}
-	function isOpen(id: string) {
-		return value.includes(id);
-	}
-
-	// Effect
-	useEffect(() => {
-		onValueChange(value);
-	}, [onValueChange, value]);
-
-	// Context
-	const ctx = {
-		animDuration,
-		iconOpen,
-		iconClosed,
-		open,
-		close,
-		toggle,
-		isOpen
-	};
+	// Zag
+	const [state, send] = useMachine(
```

Comment:
Just for consistency, I noticed that sometimes the first variable is named `state` and sometimes `snapshot`. I'd opt for one term for all useMachine() variables

Reply:
This matches Zag's implementation instructions. I noticed it too. We'll need to investigate if this is intentional.